grails-app/services/org/broadinstitute/orsp/StorageProviderService.groovy

```diff
@@ -208,12 +209,14 @@ class StorageProviderService implements Status {
                 creationDate: new Date()
         )
         if (saveStorageDocument(document, file.getInputStream())) {
-            persistenceService.saveEvent(
-                    document.projectKey,
-                    document.creator,
-                    "Adding document " + document.fileName + " to project",
-                    null
-            )
+            if (document.projectKey != null) {
+                persistenceService.saveEvent(
+                        document.projectKey,
+                        document.creator,
+                        "Adding document " + document.fileName + " to project",
+                        EventType.UPLOAD_DOCUMENT
+                )
+            }
```

Comment:
So  this is a good compromise in that we only save events for documents uploaded directly for an issue/consent group. We need some kind of TODO/ticket to follow up on saving an event for when a CCL document is uploaded too.